data/json/item_actions.json

```diff
@@ -689,6 +689,11 @@
     "id": "PORTAL",
     "name": { "str": "Place" }
   },
+  {
+    "type": "item_action",
+    "id": "POSTABLE",
```

Comment:
Nitpick, but I'd rename this from adjective-form to verb-form or noun-form. Adjective-forms are good for json flags and sound strange for _actions_, despite it's only an id of an action.

Reply:
Totally fine, I initially implemented it as a JSON flag and didn't change the name afterwards. I'll make it "POST_UP"